jupyter_client/kernelapp.py

```
@@ -2,87 +2,92 @@
         self.log.info("To connect a client: --existing %s", os.path.basename(cf))
 
     def _record_started(self) -> None:
-        """For tests, create a file to indicate that we've started
-
-        Do not rely on this except in our own tests!
-        """
         fn = os.environ.get("JUPYTER_CLIENT_TEST_RECORD_STARTUP_PRIVATE")
-        if fn is not None:
-            with open(fn, "wb"):
-                pass
+        if fn != None: 
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)